jenkins/osa_onmetal.functions.groovy

```diff
@@ -438,31 +438,42 @@ def aggregate_parse_failed_smoke(host_ip, results_file, elasticsearch_ip) {
     '''
     """
 
-	if (results_file == 'after_upgrade') {
-	    //Pull persistent, during, api, smoke results from onmetal to ES 
-	    sh """
-            ssh -o StrictHostKeyChecking=no ubuntu@${elasticsearch_ip} '''
-	    elastic-upgrade -u /home/ubuntu/output/api.uptime.out -d /home/ubuntu/output/during_output.txt -p /home/ubuntu/output/persistent_resource.txt -b /home/ubuntu/subunit/smoke/before_upgrade -a /home/ubuntu/subunit/smoke/after_upgrade
-	    ''''
-	    """
-	}
-	else {
-	    sh """
-            ssh -o StrictHostKeyChecking=no ubuntu@${elasticsearch_ip} '''
-	    elastic-upgrade -b /home/ubuntu/subunit/smoke/before_upgrade
-	    '''
-	    """
-	}
+    if (results_file == 'after_upgrade') {
+        //Pull persistent, during, api, smoke results from onmetal to ES 
+        sh """
+        ssh -o StrictHostKeyChecking=no ubuntu@${elasticsearch_ip} '''
+        elastic-upgrade -u /home/ubuntu/output/api.uptime.out -d /home/ubuntu/output/during.uptime.out -p /home/ubuntu/output/persistent_resource.txt -b /home/ubuntu/subunit/smoke/before_upgrade -a /home/ubuntu/subunit/smoke/after_upgrade
+        ''''
+        """
+    }
+    else {
+        sh """
+        ssh -o StrictHostKeyChecking=no ubuntu@${elasticsearch_ip} '''
+        elastic-upgrade -b /home/ubuntu/subunit/smoke/before_upgrade
+        '''
+        """
+    }
 
 }
 
-
-def parse_results() {
+def install_elastic_parser() {
 	
     sh """
+    rm -rf elastic-benchmark
     git clone https://github.com/osic/elastic-benchmark
     sudo pip install -e elastic-benchmark
```

Comment:
Why are you installing the package in development mode? This should be installed as an official package now:
pip install elastic-benchmark/